lapis2/src/test/kotlin/org/genspectrum/lapis/model/AggregatedModelTest.kt

```diff
@@ -99,6 +113,45 @@ class AggregatedModelTest {
                     ),
                 ),
             ),
+            Arguments.of(
+                mapOf(
+                    "nucleotideMutations" to "G123A",
+                ),
+                And(
+                    listOf(
+                        And(
+                            listOf(
+                                NucleotideSymbolEquals(123, "A"),
+                            ),
+                        ),
+                    ),
+                ),
+            ),
+            Arguments.of(
+                mapOf(
+                    "nucleotideMutations" to "G123A,567T",
+                ),
+                And(
+                    listOf(
+                        And(
+                            listOf(
+                                NucleotideSymbolEquals(123, "A"),
+                                NucleotideSymbolEquals(567, "T"),
+                            ),
+                        ),
+                    ),
+                ),
+            ),
+        )
+
+        @JvmStatic
+        fun getNucleotideMutationWithWrongSyntax() = listOf(
+            Arguments.of("A123"),
+            Arguments.of("123"),
```

Comment:
These two are entirely valid! They look for sequences that have any mutation at position 123.

See https://cov-spectrum.org/about "How can I search a variant?"

Reply:
Good catch. I added tests, that show that both work. As far as I can tell, Silo currently does not support "any mutation" besides using the symbol `N` (which I used as a workaround), but this not the intended behavior, is it?